crates/bevy_remote/Cargo.toml

```diff
@@ -26,6 +26,7 @@ bevy_platform = { path = "../bevy_platform", version = "0.16.0-dev", default-fea
   "std",
   "serialize",
 ] }
+bevy_asset = { path = "../bevy_asset", version = "0.16.0-dev" }
```

Comment:
Should this be given a feature and made optional? I feel like it shouldn't be required to have to pull in `bevy_asset` when all you want is the ECS for example.